### PR TITLE
Firebase admin fix

### LINKED ISSUE #109: UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory
To reproduce (in an empty directory):

```sh
yarn add firebase-admin
yarn add --dev @zeit/ncc
```

`index.js`
```js
require('firebase-admin')

```

Run `ncc`

```sh
node_modules/.bin/ncc build index.js
```

Result:

```
Emitting directory ~/tmp/firebase-dependencies/node_modules/@google-cloud/firestore/build/protos as protos
ncc: Module directory "~/tmp/firebase-dependencies/node_modules/hash-stream-validation" attempted to require "fast-crc32c" but could not be resolved, assuming external.
ncc: Module directory "~/tmp/firebase-dependencies/node_modules/bytebuffer/dist" attempted to require "memcpy" but could not be resolved, assuming external.
ncc: Module directory "~/tmp/firebase-dependencies/node_modules/debug/src" attempted to require "supports-color" but could not be resolved, assuming external.
ncc: Module directory "~/tmp/firebase-dependencies/node_modules/https-proxy-agent/node_modules/debug/src" attempted to require "supports-color" but could not be resolved, assuming external.
(node:17208) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '~/tmp/firebase-dependencies/dist/protos/firestore_proto_api.d.ts'
    at Object.openSync (fs.js:436:3)
    at Object.writeFileSync (fs.js:1187:35)
    at module.exports.ncc.then (~/tmp/firebase-dependencies/node_modules/@zeit/ncc/dist/ncc/cli.js:2410:14)
(node:17208) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:17208) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

### PR DESCRIPTION
**Pull request by guybedford**
Fixes #109, as well as improving the CLI error messages.

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -176,3 +177,8 @@ switch (args._[0]) {
     console.error(`Error: Invalid command "${args._[0]}"\n${usage}`);
     process.exit(1);
 }
+
+// remove me when node.js makes this the default behavior
+process.on("unhandledRejection", e => {
+  throw e;
+});
```
Comment (rauchg) [inline src/cli.js]: Where is this coming from? We should always try to handle the promises we create. Agreed that it's a good thing to add as a safeguard

Comment (guybedford) [reply src/cli.js]: Good point - I've handled the rejection rather.

Comment (rauchg) [reply src/cli.js]: I do like the safeguard. I strongly dislike Node's default mechanism

Comment (guybedford) [reply src/cli.js]: Sure added it back too!

**Pull request merged**